backend/app/Models/User.php

```diff
@@ -62,4 +64,78 @@ public function answers()
     {
         return $this->hasMany(Answer::class);
     }
+
+    public static function getProfileDetails($userId)
+    {
+        $user_details = self::where('isAdmin', 0)
+                            ->where('id', $userId)
+                            ->first();
+
```

Comment:
For user ra mn ni noh? Dapat guro ta mag return 404 if and id ka wala ni exist so instead of `first()` lang pd ni i `firstOrFail()` para if wala gni sulod diretso na siya retun `not found`

Reply:
Yes sir, will update it to `firstOrFail()` now